source/source.go

```diff
@@ -50,14 +50,33 @@ func (s *Source) Configure(ctx context.Context, cfg map[string]string) error {
 	return nil
 }
 
-func (s *Source) Open(ctx context.Context, _ sdk.Position) (err error) {
+func (s *Source) Open(ctx context.Context, sdkPos sdk.Position) (err error) {
 	s.svc, err = common.NewSQSClient(ctx, s.config.Config)
 	if err != nil {
 		return fmt.Errorf("failed to create source sqs client: %w", err)
 	}
 
+	sdk.Logger(ctx).Info().Msg("connected to sqs")
+
+	queueName := &s.config.AWSQueue
+	if sdkPos != nil {
+		pos, err := common.ParsePosition(sdkPos)
```

Comment:
Left a [similar comment](https://github.com/conduitio-labs/conduit-connector-sqs/pull/125/files#r1701961673) on the other PR, but I think these methods would make more sense if they were under a different package.